docs/clipboard-unsanitized/explainer.md

```diff
@@ -88,61 +104,28 @@ EndFragment:00000463
 In standard html format, Safari inserts both sanitized & unsanitized version of html content. It inserts the html content provided in the setData API into the clipboard using a custom webkit format type(`com.apple.Webkit.custom-pasteboard-data`). When `getData` is called, the HTML content in the custom webkit format type is returned (makes round tripping possible).
```

Comment:
Does Safari always return data from `com.apple.Webkit.custom-pasteboard-data` when HTML is pasted? My understanding is that they only do that for same-origin sites.

Reply:
Ah missed that important bit of information here :) Thanks!